pebble_tool/commands/sdk/project/convert.py

```diff
@@ -82,25 +83,20 @@ def _ignore_npm(self):
     def _convert_project(self):
         project_root = os.getcwd()
         project_template_path = os.path.join(self.get_sdk_path(), 'pebble', 'common', 'templates')
-        if not os.path.exists(project_template_path):
-            project_template_path = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'sdk', 'templates')
+        tool_project_template_path = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'sdk', 'templates')
 
         self._generate_appinfo_from_old_project(project_root)
 
         wscript_path = os.path.join(project_root, "wscript")
 
-        wscript2_hash = hashlib.md5(open(os.path.join(project_template_path, 'wscript_sdk2')).read()).hexdigest()
-        wscript3_hash = hashlib.md5(open(os.path.join(project_template_path, 'wscript')).read()).hexdigest()
-        with open(wscript_path, "r") as f:
-            current_hash = hashlib.md5(f.read()).hexdigest()
-
-        if wscript2_hash != current_hash and wscript3_hash != current_hash:
-            print('WARNING: You had modified your wscript and those changes will be lost.\n'
-                  'Saving your old wscript in wscript.backup.')
-            os.rename(wscript_path, wscript_path + '.backup')
+        print('Saving your old wscript in wscript.backup.')
+        os.rename(wscript_path, wscript_path + '.backup')
 
         print('Generating new 3.x wscript')
-        copy2(os.path.join(project_template_path, 'wscript'), wscript_path)
+        try:
```

Comment:
I started down that path (https://github.com/pebble/pebble-tool/tree/feature/update-convert-project), but then got tied up in working out how to move files around so temporarily abandoned the more involved effort. 

We can't use the wscript specified currently by templates.json because it expects the file structure of a project to be different. I just grabbed the latest 3.x wscript file and "archived" it forever as "wscript" in the pebble-tool